### PR TITLE
1994 Fix bdd tests configuration

### PR DESCRIPTION
**Pull request by AkioFaris**
#1994 
merge after #1996

### CODE REVIEW

**Comment by mf-code** [comment]
[5316] - eKdSwwvn

**Comment by mf-code** [comment]
[5316] - oxi3qv9r

**Comment by mf-code** [comment]
[5316] - zxwR5liR

**Comment by mf-code** [comment]
[5316] - Allure report on Netlify: https://5ebe9c2c33a14027878c2862--musing-swartz-6fc060.netlify.app

**Comment by AkioFaris** [comment]
Do not merge yet: BDD tests report is not generated.

**Comment by mf-code** [comment]
[5329] - k487IXYz

**Comment by mf-code** [comment]
[5329] - kpJjYmCS

**Comment by mf-code** [comment]
[5329] - Allure report on Netlify: https://5ec2524c2723aaf37ca5ea5e--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5331] - ttyVQfTn

**Comment by mf-code** [comment]
[5331] - OR1S82wX

**Comment by mf-code** [comment]
[5331] - Allure report on Netlify: https://5ec256559a08505b10e50967--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5333] - VjWbBEtT

**Comment by mf-code** [comment]
[5333] - F5B2TDR6

**Comment by mf-code** [comment]
[5333] - Allure report on Netlify: https://5ec26ac658e87866c94bb9e5--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5334] - QHbxkNmR

**Comment by mf-code** [comment]
[5334] - 4uF33pJ3

**Comment by mf-code** [comment]
[5334] - Allure report on Netlify: https://5ec274cf11ca370176ed9e19--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5336] - 7CvJ2ReC

**Comment by mf-code** [comment]
[5336] - arqTMaXU

**Comment by mf-code** [comment]
[5336] - Allure report on Netlify: https://5ec2878a8e4f1d279235a72b--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5337] - hEXxPDlg

**Comment by mf-code** [comment]
[5337] - FSlAQMVy

**Comment by mf-code** [comment]
[5338] - lCmMaURX

**Comment by mf-code** [comment]
[5338] - 0dhsHGAf

**Comment by mf-code** [comment]
[5338] - kf3QYNpP

**Comment by mf-code** [comment]
[5338] - Allure report on Netlify: https://5ec297a93c970185f0cf7ca5--musing-swartz-6fc060.netlify.app

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.steps;
+package cucumberTests.steps;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           

Complexity increasing per file
==============================
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java  1
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java  1
- jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java  5
- test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/steps/UserStepdefs.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411&bid=18135429)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.test.data;
+package cucumberTests.test.data;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/TestData.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

File: jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests;
+package cucumberTests;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java [hunk 4]
```diff
@@ -1,20 +1,20 @@
-package cucmberTests;
+package cucumberTests;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/TestRunner.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

File: jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.stepdefs;
+package cucumberTests.stepdefs;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/stepdefs/UserStepdefs.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

File: test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package cucmberTests.test.data;
+package cucumberTests.test.data;
```
Comment (codacy-review) [inline test-examples/jdi-bdd-no-po-tests/src/test/java/cucumberTests/test/data/User.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5532411)

**Comment by mf-code** [comment]
[5339] - gsTaSoZT

**Comment by mf-code** [comment]
[5339] - C0xdBDK7

**Comment by mf-code** [comment]
[5339] - Y5uIWs3o

**Comment by mf-code** [comment]
[5339] - ztm32Ocz

**Comment by mf-code** [comment]
[5339] - Allure report on Netlify: https://5ec2a59111ca377085ed98ed--musing-swartz-6fc060.netlify.app

**Pull request merged**